### PR TITLE
README.md: about enhancements

### LINKED ISSUE #1: Trophies
There is no need for trophies as its analysis is already described in cards and the GitHub profile. Therefore, Trophies are not needed as they represent duplicated analysis in an uncleared way

### LINKED ISSUE #2: linkedIn link is unavailable
as it goes to https://www.linkedin.com/in/https:/www.linkedin.com/in/thomas-meshail/ and it should only be https:/www.linkedin.com/in/thomas-meshail/

### PR DESCRIPTION
**Pull request by pavly-gerges**
Simple is Perfect !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +22,6 @@
...
 
-<p><img align="left" src="https://github-readme-stats.vercel.app/api/top-langs?username=2math0&show_icons=true&locale=en&layout=compact" alt="2math0" /></p>
-
-<p>&nbsp;<img align="center" src="https://github-readme-stats.vercel.app/api?username=2math0&show_icons=true&locale=en" alt="2math0" /></p>
-
-<p><img align="center" src="https://github-readme-streak-stats.herokuapp.com/?user=2math0&" alt="2math0" /></p>
-
+| My Love | Git Statistics |
```
Comment (2Math0) [inline README.md]: no need to put them in table

**Comment by pavly-gerges** [comment]
@2Math0 Could you please link those issues: 
- [x] #1 
- [x] #2 

This PR resolves them.

**Review by 2Math0 — CHANGES_REQUESTED** [review]
code is not readable

File: README.md (hunk 1 above)
Comment (pavly-gerges) [reply README.md]: Okay your new changes look good !

**Review by 2Math0 — COMMENTED** [review]
Amazing

**Pull request merged**